tests/conftest.py

```diff
@@ -13,7 +13,7 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-from distutils.version import LooseVersion
+import packaging.version
```

Comment:
S: from packaging.version import Version, and then change the below, but YMMV

Reply:
We would need to import also `InvalidVersion`, so I think is is cleaner this way TBH. (Another not too good reason is that I basically copied from [here](https://github.com/mendersoftware/meta-mender/commit/6d2ca0800387a17fccd9b51b9f82f6c17903fd9d)).